mayland-blocks/style.css

```diff
@@ -24,20 +24,6 @@ body {
 	-moz-osx-font-smoothing: grayscale;
 }
 
-/* Set editor widths. */
```

Comment:
I branched from https://github.com/Automattic/themes/pull/3621 because I need those changes for this to look correct.